test/invoker_test.rb

```diff
@@ -163,6 +163,38 @@
     assert_match(/WARNING/, warnings)
   end
 
+  test 'should change level on logger when --log-level is specified' do
+    input = <<~'EOS'
+    skip to <<install>>
+
+    . download
+    . install[[install]]
+    . run
+    EOS
+    output = nil
+    redirect_streams do |_, err|
+      invoke_cli(%w(--log-level info), '-') { input }
+      output = err.string
+    end
+    assert_equal 'asciidoctor: INFO: possible invalid reference: install', output.chomp
+  end
+
+  test 'should not log when --log-level and -q are both specified' do
```

Comment:
If `--log-level` and `-v` are both specified, which option should prevail?
In my opinion, `--log-level` should have a higher precedence over `-v` because `--log-level` is more explicit since you explicit state a log level (whereas `-v` is an alias for `--log-level debug`).

Alternatively, we could use the option order but not sure if this is supported (i.e., last or first declared wins).

Reply:
Right you are. I had programmed this logic, but forgot to add a test for it. `-q` wins out over `--log-level`, but `--log-level` wins out over `-v`.